cypress/e2e/signIn.cy.js

```diff
@@ -3,5 +3,20 @@
 describe('Sign In page', () => {
   it('should provide an ability to log in', () => {
     
+    cy.visit('https://react-redux.realworld.io/');
+    
```

Comment:
no need to leave empty lines between every command, you can try to group commands logically and seperate groups